### PR TITLE
Iz gawb 2459 new method import

### PR DESCRIPTION
**Pull request by zarsky-broad**
This will be refactored to allow the components to be (more) context-agnostic at a later point.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/method_repo_NEW/method_repo_table.cljs [hunk 1]
```diff
@@ -63,41 +64,48 @@
...
                                :column-data :synopsis
                                :sort-by string/lower-case}
-                              {:header "Owners" :initial-width 175
+                              {:header "Owners" :initial-width 225
                                :column-data :managers
                                :as-text (partial string/join ", ")}
                               {:header "Snapshots" :initial-width 94 :filterable? false
                                :column-data :numSnapshots}
-                              {:header "Configurations" :initial-width 117 :filterable? false
+                              {:header "Configurations" :initial-width :auto :filterable? false
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/method_repo_NEW/method_repo_table.cljs:95]: `:auto` behaves poorly when the table is too wide for the view, which easily happens here.

**Review by MattPutnam — APPROVED** [review]
Code looks good, I'll investigate tests now

File: src/cljs/main/broadfcui/page/method_repo_NEW/method_repo_table.cljs (hunk 1 above)
Comment (zarsky-broad) [reply src/cljs/main/broadfcui/page/method_repo_NEW/method_repo_table.cljs:95]: I can't get it to misbehave in this instance...

**Comment by davidangb** [comment]
Functional testing:
* in the select-configuration dialog, the draggy scrollbar between the table and the config details butts up against the buttons beneath it
* on small-width screens, the config details may be totally invisible
* I'm having a hard time with the subtle visual differences between the import dialog and the export dialog - mostly about the white vs. grey background sections

**Review by davidangb — COMMENTED** [review]
no real code comments, but a few visual tweaks if possible.

File: src/cljs/main/broadfcui/components/split_pane.cljs [hunk 2]
```diff
@@ -27,6 +27,7 @@
          [:div {:style {:flex "0 0 10px"
                         :display "flex" :flexDirection (if left "column" "row") :justifyContent "center"
                         :backgroundColor (:background-light style/colors)
+                        :margin (str "0 " slider-padding)
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/components/split_pane.cljs]: The slider is designed to work both vertically and horizontally, so you should check which sides to apply the padding:

```clojure
(if top (str slider-padding " 0") (str "0 " slider-padding))
```

**Comment by davidangb** [comment]
jenkins retest

**Pull request merged**